### DISCUSSION TITLE
30N30P Multi-element airfoil

### DISCUSSION THREAD

**Discussion by billdafoam** [mdolab/dafoam#638] [General]
Hi there, 
I am trying to run the 30N30P three element case and the following message appears. Could you please tell me or send the respective latest script on purpose? Please see the image below for assistance and consideration . 
[Fata_error_ 30N30P_airfoil.docx](https://github.com/mdolab/dafoam/files/15309061/Fata_error_.30N30P_airfoil.docx)
I look forward for any supportive answer.
Cheers

**Comment by friedenhe** [comment]
I would need more information to debug. I just ran the case without any problems by doing `./preProcessing.sh && python runScript.py` What did you do before seeing the error?

**Comment by billdafoam** [comment]
Cheers indeed for your prompt response, but I still present the same issues after my effort in running the test case. I am wondering if the vsrsion of DAFOAM 3.1.1 is still compatible with the respective DOCKER'S 3.1.1. See I am trying to run the case using DAFOAM 3.0.0 version please. Should I try to use the DAFOAM 3.1.1 updated version instead?  I have not amended any of the input files, also find attached the file with the displayed  images. I look forward to hearing from you ASAP.  Regards 
[DAFOAM_IMAGES_30N30P_MULTIELEMENT.docx](https://github.com/mdolab/dafoam/files/15331185/DAFOAM_IMAGES_30N30P_MULTIELEMENT.docx)

**Comment by billdafoam** [comment]
Kindly also find attached the respective. logOpt.txt file on purpose for a better consideration and I look forward to hearing from you. 
Cheers 
[logOpt.txt](https://github.com/mdolab/dafoam/files/15331259/logOpt.txt)

**Comment by friedenhe** [comment]
Did you see any error when running `./preProcessingh.sh`?

**Reply by billdafoam** [reply]
Not really, the error message appeared when the I tried to run it via python runScript.py sir. 
Does it play any role if the DAFOAM version is the 3.0.0, whilst the docker's version is 3.1.1 instead ?
Please let me know, if I should redownload the DaFoam 3.1.1 version instead. 
Cheers

**Comment by friedenhe** [comment]
> Please let me know, if I should redownload the DaFoam 3.1.1 version instead.

Yes
